src/admin/user/user.controller.ts

```diff
@@ -35,8 +36,12 @@ export class UserController {
   }
 
   @Get()
```

Comment:
Add ApiQuery configuration for these parameters (everything should be optional).